### DISCUSSION TITLE
How to change size of `ProgressRing` ?

### DISCUSSION THREAD

**Discussion by andrew-ares** [bdlukaa/fluent_ui#945] [Q&A] [answered]
I want to use `ProgressRing` as a child of `FilledButton`, but it looks oversize.

![image](https://github.com/bdlukaa/fluent_ui/assets/40336192/88e0104f-6c22-4e8b-8f64-090fff960706)

**Comment by bdlukaa** [comment, marked as answer]
Hello! You can adjust its size by wrapping it in a `SizedBox`. You may also want to set `ProgressRing.strokeWidth`

**Additional resources**
* https://github.com/bdlukaa/fluent_ui/blob/master/example/lib/screens/home.dart#L78-L79
* https://pub.dev/documentation/fluent_ui/latest/fluent_ui/ProgressRing/strokeWidth.html
